packages/arb-util/configuration/configuration.go

```diff
@@ -567,10 +610,34 @@ func endCommonParse(k *koanf.Koanf) (*Config, *Wallet, error) {
 	}
 	err := k.UnmarshalWithConf("", &out, koanf.UnmarshalConf{DecoderConfig: &decoderConfig})
 	if err != nil {
-
 		return nil, nil, err
 	}
 
+	if len(out.Fireblocks.APIKey) != 0 || len(out.Fireblocks.BaseURL) != 0 ||
+		len(out.Wallet.FireblocksPrivateKey) != 0 || len(out.Fireblocks.SourceAddress) != 0 ||
+		len(out.Fireblocks.SourceId) != 0 || len(out.Fireblocks.SourceType) != 0 {
```

Comment:
Could you do `if out.Fireblocks != (Fireblocks{})`?

Reply:
The idea is to catch an error such that if any field is set, all the fields should be set.